extensions/wikia/Recirculation/js/helpers/blacklist.js

```diff
@@ -0,0 +1,59 @@
+define('ext.wikia.recirculation.helpers.blacklist', [
+    'wikia.cache',
+    'wikia.window',
+    'wikia.cookies'
+], function (cache, window, cookies) {
+    'use strict';
+
+    var CACHE_KEY = 'recommendations_blacklist';
```

Comment:
Is there no way to share logic for this between mobile-wiki and app?

Reply:
unfortunately not.

- different approaches to caching (localStorageConnector / wikia.cache)
- require/ES modules

and extracting such small module to separate repo did not make sense for me